src/app/search/search-notebook-table/search-notebook-table.component.html

```diff
@@ -13,8 +13,20 @@
               notebook?.source.repository +
               (notebook?.source.workflowName ? '/' + notebook?.source.workflowName : '')
           }}</a>
-          <div class="truncate-text-2 size-small" *ngIf="notebook.source.topicAutomatic" [matTooltip]="notebook.source.topicAutomatic">
-            {{ notebook.source.topicAutomatic }}
+          <div class="truncate-text-2 size-small" [matTooltip]="notebook?.source.topicAutomatic" matTooltipPosition="left">
```

Comment:
This block is repeated several times times; you could make a component or template out of it.

Reply:
created a new component